File: Python/ejercicio2Fich/Ejercicio2.py
'''
Realiza un programa que lea el fichero creado en el ejercicio anterior y que
muestre los números por pantalla.
'''

# Abre archivo en modo lectura
archivo = open('primos.dat','r')

while True:
    linea = archivo.readline()  # lee línea
    if not linea:
        break  # Si no hay más se rompe bucle
    print(linea)  # Muestra la línea leída
archivo.close()

File: Java/ejercicio2Fich/Ejercicio2.java
package ejercicio2Fich;

import java.io.*;

/**
 * Realiza un programa que lea el fichero creado en el ejercicio anterior y que
 * muestre los números por pantalla.
 */

public class Ejercicio2 {

	public static void main(String[] args) {
		try {
			BufferedReader br = new BufferedReader(new FileReader("primos.dat"));
			String linea = "";
			while (linea != null) {
				System.out.println(linea);
				linea = br.readLine();
			}
			br.close();
		} catch (Exception e) {
			System.err.println("Fallo");
		}

	}

}
